doc/services/debugging/gdbstub.rst

```diff
@@ -111,63 +130,63 @@ for its implementation as a Twister test.
 
       .. code-block:: text
 
-         (gdb) file <build directory>/zephyr/zephyr.elf
+         (gdb) symbol-file <build directory>/zephyr/zephyr.elf
 
       Response from GDB:
 
       .. code-block:: text
 
          Reading symbols from <build directory>/zephyr/zephyr.elf...
 
-   #. Tell GDB to connect to the server:
+   #. Tell GDB to connect to the Zephyr gdbstub serial backend which is exposed
+      earlier as a server through the TCP port ``-serial`` redirection at QEMU.
```

Comment:
Not sure this works better than the wording I had proposed (ex. I think "at QEMU" isn't proper English)